core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
LGTM. Would be cool to get this merged tonight, so that I can build on top of that over the weekend. What do you think @vkostyukov?